Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp

```diff
@@ -1936,7 +1953,7 @@ void ResourceLoadStatisticsStore::logFrameNavigation(const RegistrableDomain& ta
         scheduleStatisticsProcessingRequestIfNecessary();
 }
 
-void ResourceLoadStatisticsStore::logCrossSiteLoadWithLinkDecoration(const NavigatedFromDomain& fromDomain, const NavigatedToDomain& toDomain)
+void ResourceLoadStatisticsStore::logCrossSiteLoadWithLinkDecoration(const NavigatedFromDomain& fromDomain, const NavigatedToDomain& toDomain, DataRemovalFrequency newDataRemovalFrequency)
```

Comment:
I think the parameter should be DidFilterKnownLinkDecoration not DataRemovalFrequency, because this would never be DataRemovalFrequency::Never right?